src/extensions/score_metamodel/checks/check_options.py

```diff
@@ -33,6 +33,28 @@ def get_need_type(needs_types: list[ScoreNeedType], directive: str) -> ScoreNeed
     raise ValueError(f"Need type {directive} not found in needs_types")
 
 
+def _normalize_values(raw_value: str | list[str] | None) -> list[str]:
+    """Normalize a raw value into a list of strings."""
+    if isinstance(raw_value, str):
+        return [raw_value]
+    if isinstance(raw_value, list) and all(isinstance(v, str) for v in raw_value):
+        return raw_value
+    return [str(raw_value)]
```

Comment:
Not sure about that last part here... how about 
```suggestion
    raise ValueError("unexpected types")
```